### ISSUE TITLE
Relocate user 'fido'

### ISSUE THREAD

**Issue by 01micko** [puppylinux-woof-CE/woof-CE#615]
Fido is a bit of a mess. It really should be disabled until it can be fixed, but I'm not saying 'do that'; it is unnecessary work.

Fido is a lot different to Spot. Spot is the 'system' account for apps to be run as non-root. It mostly works fine.

What really needs to happen is for the Fido account to be relocated to /home. This doesn't necessarily have to happen on the ISO. It would be trivial to copy all of /root (filtered of course) to /home/fido, chown everything and voila! Most of the stuff is in place for a non-root user to get stuff done. Of course things will be missed and lots of pets wont work well but the alternative is to weed out fido all together. I think this can go hand in hand with the Xorg fixup that is badly needed re #583 .

This is the least invasive approach I feel. Puppy users love root! (Self included - partly why I love slackware too - you can run root as much as you like).

**Comment by wdlkmpx** [comment]
I'm using fido for the first time. It was an accident as i was just testing what root2user does, then rebooted.

It looks like i'm trapped, i can't su with the default password, askpass has some incorrect code and sudo segfaults, don't know why

I can see why it should be moved to /home/fido, although i think it should be renamed to 'uzer' or something more mainstream..

**Comment by wdlkmpx** [comment]
su does work, but sudo keeps segfaulting.. wrong version

**Comment by wdlkmpx** [comment]
With  the changes i made i have a minimal stable and working system under fido (it doesn't include installing pet pkgs). I'm mainly testing with pcmanfm + pvm0.1.17.. and i have no issues mounting or unmounting anything. i do see issues in the rox desktop but i couldn't care less about it, but most of them should be fixed anyway.

However relocating fido seems to be next step, and this will certainly break many other things and all pet pkgs with hardcoded /root. But i don't mind that.

However i really don't like fido, i'd like to perform a massive sed to replace it with another name. Mostly because I have 5 cats and i don't like dogs because they usally bark me at when i'm walking in the streets. That's why (and the 2 barks when X starts). i partially  don't feel connected to the project..

So I think it's time to disable fido in shutdownconfig.

**Comment by peabee** [comment]
This is my first awareness that pvm-0.1.17 exists......
Have you made a .pet? Can you make it available for others to test/use?
There is a forum thread......

**Comment by ninaholic** [comment]
Hi, I found a more extensive thread about this topic here:
http://www.murga-linux.com/puppy/viewtopic.php?t=71358

I think fido worked in Slacko 6.9.6.4? Barry says keeping it in root is ok and less work. 😆

**Comment by wdlkmpx** [comment]
https://github.com/01micko/pup-volume-monitor

release tarball created with fido:
https://github.com/wdlkmpx/pup-volume-monitor/releases/download/0.1.17/pup-volume-monitor-0.1.17.tar.gz

last time i tried to post a message was to announce that and invite developers to keep fixing it, but my account was suspended for being a cool and funny guy. haven't tried to login since then

**Comment by wdlkmpx** [comment]
Yeah I actually get Barry's point. fido being in /root is a quick and working solution. It was certainly working in slacko 6.x, but several scripts had bugs including ask_pass, it was mostly broken for several operations with gui and completely Broken for the stuff i use.

Moving fido is kind of a radical change that defeats the Puppy awfully bad design it's famous for and requires way more work and people outside woof to fix their scripts too. Maybe it's better to keep fido in /root and create another experimental user for this purpose: blue923 in a new branch "w" (where the story ends).

**Comment by wdlkmpx** [comment]
xdg compliance makes this even easier.

At this point this is feasible. And I think spot is the user, fido also known as finn does not qualify unless it's moved to /home/finn

As I said a year ago, I made it work with rox and pcmanfm. But it doesn't mean it will work the same in other compat distro versions, it might depend on how "close" a build is to the default woofce stuff. It's that simple. It's not hard to understand.

So this should happen. I see I added 2 commits 2 years ago, lost commits of course, but I guess I can add them again.

It's easier now. The way fido (also known as finn) is implemented is just `wrong`. 

Users should not share the same home folder, of course they will end up having different stuff, but that's how it works.

But with xdg compliance woofce offers a mostly usable desktop to all users by default .. does it sound nice? It does. wooo
